### PR TITLE
import_spss informative error messages, closes #8

### LINKED ISSUE #8: More informative error message labeledStrings in import_spss()
Point user to the `labeledStrings` argument.

### PR DESCRIPTION
**Pull request by beckerbenj**
Users should be hinted to help page.

### CODE REVIEW

File: R/char_valLabels2numeric.R [hunk 1]
```diff
@@ -25,13 +25,13 @@ char_valLabels2numeric.savDat<- function(rawDat, labeledStrings) {
...
       warning("Some values with value labels or missing tags of variable ", char_var,
-              " cannot be coerced to numeric. This possibly corrupts all meta data.")
+              " cannot be coerced to numeric. This possibly corrupts all meta data. For other import behavior check out the 'labeledStrings' argument.")
     }
 
     if(identical(labeledStrings, "drop")) {
       #browser()
       warning("Some values with value labels or missing tags of variable ", char_var,
-       " cannot be coerced to numeric and are therefore changed to NA.")
+       " cannot be coerced to numeric and are therefore changed to NA. For other import behavior check out the 'labeledStrings' argument.")
```
Comment (sachseka) [inline R/char_valLabels2numeric.R:34]: this works!

File: R/char_valLabels2numeric.R [hunk 2]
```diff
@@ -25,13 +25,13 @@ char_valLabels2numeric.savDat<- function(rawDat, labeledStrings) {
 
     if(identical(labeledStrings, "keep")) { ## corresponding behavior in extract_labels has to be adapted!
       warning("Some values with value labels or missing tags of variable ", char_var,
-              " cannot be coerced to numeric. This possibly corrupts all meta data.")
+              " cannot be coerced to numeric. This possibly corrupts all meta data. For other import behavior check out the 'labeledStrings' argument.")
     }
```
Comment (nickhaf) [inline R/char_valLabels2numeric.R:29]: great idea!

**Review by sachseka — APPROVED** [review]

**Review by nickhaf — APPROVED** [review]
looking good

File: R/char_valLabels2numeric.R (hunk 1 above)
Comment (weirichs) [reply R/char_valLabels2numeric.R:34]: LGTM

Comment (franikowsp) [inline R/char_valLabels2numeric.R:34]: Great!

**Review by franikowsp — APPROVED** [review]
👌

File: R/char_valLabels2numeric.R [hunk 3]
```diff
@@ -25,13 +25,13 @@ char_valLabels2numeric.savDat<- function(rawDat, labeledStrings) {
 
     if(identical(labeledStrings, "keep")) { ## corresponding behavior in extract_labels has to be adapted!
       warning("Some values with value labels or missing tags of variable ", char_var,
-              " cannot be coerced to numeric. This possibly corrupts all meta data.")
+              " cannot be coerced to numeric. This possibly corrupts all meta data. For other import behavior check out the 'labeledStrings' argument.")
```
Comment (weirichs) [inline R/char_valLabels2numeric.R:28]: Ja sieht gut aus, mach das!

**Review by weirichs — APPROVED** [review]

**Pull request merged**